packages/vm/core/blocklog/interface.go

```diff
@@ -344,17 +337,48 @@ func RequestReceiptFromMarshalUtil(mu *marshalutil.MarshalUtil) (*RequestReceipt
 	if ret.Request, err = iscp.RequestDataFromMarshalUtil(mu); err != nil {
 		return nil, err
 	}
+
+	if isError, err := mu.ReadBool(); err != nil {
+		return nil, err
+	} else if !isError {
+		return ret, nil
+	}
```

Comment:
nitpick: else not needed
```suggestion
	}
	if !isError {
		return ret, nil
	}
```

Reply:
That only works when we create a variable first, like this: 
```	var isError bool
	
	if isError, err = mu.ReadBool(); err != nil {
		return nil, err
	}

	if !isError {
		return ret, nil
	}```

Otherwise it complains about isError not existing at the !isError check. I think because the err != nil check next to readBool puts isError into another scope. 

I would keep it how it is, because that makes it just longer. What do you think?

Reply:
oh yeah, you're right. let's just keep as-is